app/lib/features/chat/widgets/convo_card.dart

```diff
@@ -332,7 +335,7 @@ class _SubtitleWidget extends ConsumerWidget {
       case 'm.sticker':
         final body =
             eventItem
-                .msgContent()
+                .message()
                 .expect('m.sticker should have msg content')
                 .body();
```

Comment:
Also this `expect()` should be fixed in according to [your comment](https://github.com/acterglobal/a3/pull/2790#discussion_r2037752638)?